frontend/src/components/UserMailboxAuthorizations.vue

```diff
@@ -0,0 +1,101 @@
+<template>
+  <v-card  v-if="clientsWithRoles.length > 0" outlined class="subgroup">
+    <h2 class="sfds-authorizations-header">Mailbox Authorizations</h2>
+
+    <div class="nav-tabs" style="width: 100%">
+      <a
+      v-for="(item,index) in clientsWithRoles" v-bind:key="item.name"
+      :class="(selectedClient > index || selectedClient < index)? 'tabActive': 'tabInactive'"  
+      @click="selectedClient = index">
+      {{item.name}}
+      </a>
+    </div>
+    <user-update-sfds-auth
+      :selectedClient="this.clientsWithRoles[selectedClient]"
+    ></user-update-sfds-auth>
+  </v-card>
+</template>
+
+
+<script>
+import UserUpdateSfdsAuth from "@/components/UserUpdateSfdsAuth";
+import UsersRepository from "../api/UsersRepository";
+import ClientsRepository from "@/api/ClientsRepository";
+
+export default {
+  name: "UserMailboxAuthorizations",
+  props: ["userId"],
+
+  components: {
+    UserUpdateSfdsAuth,
+  },
+  data() {
+    return {
+      MailboxClientNames: ["SFDS", "HSCIS"],
+      clientsWithRoles: [],
+      selectedClient: 0,
+    };
+  },
+  async created() {
+    this.$root.$refs.UserMailboxAuthorizations = this;
```

Comment:
What's the purpose of this line?

Reply:
when the user role gets updated, they call UserMailBoxAuth to update & see if user has roles to view mailbox